docs/networking/dns/configure-your-linode-for-reverse-dns/index.md

```diff
@@ -5,13 +5,13 @@ author:
 description: 'Reverse DNS (rDNS) resolves an IP address to the designated domain name. This guide will teach you how to set it up.'
 keywords: ["reverse", "dns", "PTR"]
 license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
-aliases: ['networking/Setting-Up-Reverse-DNS-Lookup/','dns-guides/configuring-dns-with-the-linode-manager/','networking/configure-your-linode-for-reverse-dns/','networking/dns/setting-reverse-dns/','dns-manager/']
+aliases: ['networking/dns/configure-your-linode-for-reverse-dns-classic-manager-classic-manager/','dns-manager/','networking/configure-your-linode-for-reverse-dns/','networking/dns/setting-reverse-dns/','dns-guides/configuring-dns-with-the-linode-manager/','networking/setting-up-reverse-dns-lookup/']
```

Comment:
got a double `classic-manager` here:

`networking/dns/configure-your-linode-for-reverse-dns-classic-manager-classic-manager`

Reply:
fixed